lib/Controller.js

```diff
@@ -15,8 +16,10 @@ var Q = require('q');
 var qio = require('q-io/fs');
 var path = require('path');
 var util = require('util');
-var adapter = require('./Adapters/AdapterFactory.js').getAdapter();
+
 var MonitorServer=require('./MonitorServer.js');
+const events = require('events');
+
```

Comment:
Guy, there is one change that is required, unless I add the line to App.js...
var adapter = require('./Adapters/AdapterFactory.js').getAdapter(); moved to the top of Controller.js so the configuration updates will take place before any module get config.